### PR TITLE
MLOPS-356 Remove keto

### PR DESCRIPTION
**Pull request by bernardolk**

### CODE REVIEW

File: modules/ory/oathkeeper/main.tf [hunk 1]
```diff
@@ -1,14 +1,14 @@
 locals {
   config_mount_path = "/etc/config"
   secrets_mount_path = "/etc/secrets"
+  access_rules_path = "${path.module}/access-rule-oathkeeper.yaml"
 }
 
 data "template_file" "oathkeeper-access-rules"{
-  template = file("${path.module}/access-rule-oathkeeper.yaml")
+  template = file("%{if var.access_rules_path == null}${path.module}/access-rule-oathkeeper.yaml%{else}${var.access_rules_path}%{ endif }")
```
Comment (tvinagre) [inline modules/ory/oathkeeper/main.tf:8]: Isn't it better to set `${path.module}/access-rule-oathkeeper.yaml` as default value for `var.access_rules_path` and remove the `if-else` condition?

Comment (bernardolk) [reply modules/ory/oathkeeper/main.tf:8]: But I can't access the path var in the variables file :(

Comment (tvinagre) [reply modules/ory/oathkeeper/main.tf:8]: Yeah that's. what I was afraid of, so in that case it's fine

Another solution would be using a local, and do

```
locals {
   access_rules_path = "${path.module}/access-rule-oathkeeper.yaml" ? var. access_rules_path == "" : 
        var.access_rules_path
} 
```

and default `var.access_rules_path` to `""`

Comment (bernardolk) [reply modules/ory/oathkeeper/main.tf:8]: Actually, I thought about merging the `${path.module}` part to whatever the user specifies, but that is also not great. Your way allows the user to place the file outside the module folder, which is better.

**Pull request merged**